cpg-core/src/main/java/de/fraunhofer/aisec/cpg/graph/types/TypeParser.java

```diff
@@ -319,7 +275,7 @@ private static String fixGenerics(@NotNull String type) {
        * by the elaborate type specifier and at least one more horizontal whitespace, which marks
        * that it is indeed an elaborate type and not something like structMy.
        */
-      for (String elaborate : elaboratedTypes) {
+      for (String elaborate : ((CPPLanguage) getLanguage()).getElaboratedTypeSpecifier()) {
```

Comment:
Can we add language as an additional parameter here? I want to get rid of the `getLanguage` calls in the singleton type manager.